src/filesystem.cc

```diff
@@ -1293,6 +1325,15 @@ ASFileSystem::LocalizeDirectory(
   return DownloadFolder(container, object, dest);
 }
 
+Status
+ASFileSystem::LocalizeFile(
+    const std::string& path, std::shared_ptr<LocalizedDirectory>* localized)
+{
+  return Status(
```

Comment:
Should we document in Python backend that only S3 is supported right now?

Reply:
Yes. I can do the update. Maybe we can mark the feature as experiment for now, until a more permanent solution is implemented.